src/app/components/challenge/challengeevaluation/challengeevaluation.component.spec.ts

```diff
@@ -16,7 +17,7 @@ describe('ChallengeevaluationComponent', () => {
     TestBed.configureTestingModule({
       declarations: [ ChallengeevaluationComponent ],
       providers: [ ChallengeService, ApiService, GlobalService, AuthService, EndpointsService ],
-      imports: [ HttpClientModule ]
+      imports: [ HttpClientModule, RouterTestingModule ]
```

Comment:
I don't see this change relevant to this PR.

Reply:
ok i will drop that change